### PR TITLE
removed touchscreen/touch sensitive screen from TermsErrors

### PR DESCRIPTION
**Pull request by deerskindoll**
IBM style approves the use of "touchscreen", Red Hat supp style guide doesn't comment on it

### CODE REVIEW

File: .vale/styles/RedHat/TermsErrors.yml [hunk 1]
```diff
@@ -92,8 +92,7 @@ swap:
   componentization: component-based development|component model|component architecture|shared components
   componentize: develop components
   comprised of: consist of
-  computer farm|computer ranch: server cluster
-  computer farm|computer ranch: server farm
+  computer farm|computer ranch: server cluster|serverfarm
```
Comment (aireilly) [inline .vale/styles/RedHat/TermsErrors.yml:95]: s/serverfarm/server farm

Comment (deerskindoll) [reply .vale/styles/RedHat/TermsErrors.yml:95]: sorry, I got rid of the duplicate and made a new error while at it

**Comment by aireilly** [comment]
Can you squash or drop commits as required and we will get this merged.

**Comment by deerskindoll** [comment]
> Can you squash or drop commits as required and we will get this merged.

on it

**Comment by aireilly** [comment]
Is this PR still required?

**Comment by deerskindoll** [comment]
it's not a matter of life and death. we can cancel this.

**Pull request closed without merging**